CHANGELOG.md

```diff
@@ -139,24 +144,26 @@ Please see the [Envoy documentation](https://www.envoyproxy.io/docs/envoy/latest
 
 ### Emissary-ingress and Ambassador Edge Stack
 
-- Feature: It is now possible to use both the `getambassador.io/v2` and `getambassador.io/v3alpha1`
-  apiVersions.  This is accomplished with the addition of an `emissary-ingress-apiext` Service and
-  Deployment that handles converting between the two versions for the apiserver.  This allows the
-  user to semlessly request and author resources in whichever API version is desired, and
-  facilitates easier migration from Emissary 1.x to 2.x.  Resources authored in v3alpha1 will not be
-  visible to Emissary 1.x; resources authored in v2 will be visible to both Emissary 1.x and 2.x.
+- Change: Support for the Envoy V2 API is deprecated in Emissary-ingress v2.1, and will be removed
```

Comment:
Is it intentional that we have this message in two places?

Reply:
Yes, the first is for 2.1.2 and this one is for 2.1.0.